Emulator/TNativePrimitives.cpp

```diff
@@ -179,8 +179,6 @@ TNativePrimitives::SetEmulator( TEmulator* inEmulator )
 void
 TNativePrimitives::ExecuteNative( KUInt32 inInstruction )
 {
-	static KUInt16 n[0x1000] = { 0 };
```

Comment:
What was this used for? 🤔

Reply:
I think it was used when I wrote boot progess into the screen overlay, which is no longer needed at current CPU speeds.